lib/galaxy_test/api/test_tools.py

```diff
@@ -2436,22 +2436,45 @@ def test_group_tag_selection_multiple(self, history_id):
         output_content = self.dataset_populator.get_history_dataset_content(history_id, dataset=output)
         assert output_content.strip() == "123\n456\n456\n0ab"
 
-    @skip_without_tool("cat1")
-    def test_run_deferred_dataset(self, history_id):
+    def _run_deferred(self, history_id: str, use_cached_job=False, expect_cached_job=False, include_correct_hash=True):
+        hashes: Optional[List[Dict[str, str]]] = None
+        if include_correct_hash:
+            hashes = [{"hash_function": "SHA-1", "hash_value": "2d7dcdb10964872752bd6d081725792b3f729ac9"}]
```

Comment:
You can skip the explicit type annotation with:
```suggestion
        hashes = (
            [{"hash_function": "SHA-1", "hash_value": "2d7dcdb10964872752bd6d081725792b3f729ac9"}]
            if include_correct_hash
            else None
        )
```